e2e/single-cluster/imagescan_test.go

```diff
@@ -92,3 +68,211 @@ var _ = Describe("Image Scan", Label("infra-setup"), func() {
 		})
 	})
 })
+
+var _ = Describe("Image Scan dynamic tests pushing to ttl.sh", Label("infra-setup"), func() {
+	var (
+		clonedir   string
+		k          kubectl.Command
+		gh         *githelper.Git
+		repository *git.Repository
+		assetdir   string
+		tmpRepoDir string
+		image      string
+		imageTag   string
+	)
+
+	JustBeforeEach(func() {
+		k = env.Kubectl.Namespace(env.Namespace)
+		tmpdir := GinkgoT().TempDir()
+		clonedir = path.Join(tmpdir, "clone")
+		repository = setupRepo(k, tmpdir, clonedir, tmpRepoDir)
+	})
+
+	AfterEach(func() {
+		_, _ = k.Delete("gitrepo", "imagescan")
+		_, _ = k.Delete("secret", "git-auth")
+	})
+	When("deploy imagescan setup with pre-release images", func() {
```

Comment:
nit:
```suggestion
	When("deploying imagescan setup with pre-release images", func() {
```

Reply:
changed

Reply:
For some reason this has been undone :thinking: